src/components/ContactUsForm/ContactUs.jsx

```diff
@@ -1,27 +1,88 @@
-/* eslint-disable no-unused-vars */
 import { useBackend } from '../../contexts/BackendContext';
 import { useState } from 'react';
-import { Flex, Text, Textarea, Stack, Box, Button, Checkbox } from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+import {
+  Flex,
+  Text,
+  Textarea,
+  Stack,
+  Box,
+  Button,
+  HStack,
+  Card,
+  Spacer,
+  Link,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  Image,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  NumberIncrementStepper,
+  NumberDecrementStepper,
+} from '@chakra-ui/react';
 
 const ContactUs = () => {
   const business_ID = 0;
   const { backend } = useBackend();
-  const [name, setName] = useState('business?'); // name of business
+  // const [name, setName] = useState('business?'); // name of business
   const [text, setText] = useState('');
-  const [checkedItems, setCheckedItems] = useState([false, false, false, false, false]);
+  const [checkedItems, setCheckedItems] = useState([0, 0, 0, 0, 0, 0, 0]);
+  const [showConfirmation, setShowConfirmation] = useState(false);
+  const [isFormValid, setIsFormValid] = useState(false);
+
   const formData = {
     business_id: business_ID,
     message: null,
     timestamp: new Date().toLocaleString('en-US', { timeZone: 'America/Los_Angeles' }),
     been_dismissed: false,
   };
 
-  const handleCheck = (index, newValue) => {
-    setCheckedItems(checkedItems => {
-      const newArray = [...checkedItems];
-      newArray[index] = newValue;
-      return newArray;
-    });
+  // const handleCheck = (index, newValue) => {
+  //   setCheckedItems(checkedItems => {
+  //     const newArray = [...checkedItems];
+  //     newArray[index] = newValue;
+  //     return newArray;
+  //   });
+  // };
+
+  const ConfirmationDialog = ({ isOpen, onClose }) => (
+    <Modal size="5xl" isOpen={isOpen} onClose={onClose}>
+      <ModalOverlay />
+      <ModalContent>
+        <Stack align="center">
+          <ModalHeader mt="25">
+            <Image src="src/components/ContactUsForm/fph_logo.png" boxSize="200px" alt="fph Logo" />
+          </ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <Stack align="center">
+              <Text fontSize="3xl" color="teal">
+                Supply Request Sent!
+              </Text>
+              <Text>Your supplies will be shipped in 5-7 business days.</Text>
+            </Stack>
+          </ModalBody>
+
+          <ModalFooter>
+            <Button colorScheme="blue" mr={3} onClick={onClose}>
+              Back to Home
+              <Link to={'/BusinessDonationTrackingForm'} />
```

Comment:
Rather than using a link, make the onclick for the button with a useNavigate that takes the user back to the home page